assets/chat/js/messages/ChatMessage.js

```diff
@@ -146,4 +146,20 @@ export default class ChatMessage extends ChatUIMessage {
     this.ui.classList.toggle('msg-own', isOwn);
     this.isown = isOwn;
   }
+
+  /**
+   * @param {ChatMessage} lastMessage
+   */
+  setContinued(lastMessage) {
+    this.continued =
+      lastMessage &&
+      !lastMessage.target &&
+      lastMessage.user &&
+      (!lastMessage.ignored || lastMessage.continued) && // messages should not appear as "continued" if the previous message is ignored and was the start of the thread
+      lastMessage.user.username === this.user.username;
```

Comment:
Putting the logic that determines if a message is continued feels wrong here for a variety of reasons. Maybe moving directly into `updateMessages()` instead? It makes this method work a little more predictably too: as the setter of a boolean property.